index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Donice Portfolio</title>
+
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet" />
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+  <header>
+    <img src="./images/SideImage.svg" id="nav-image" alt="" />
```

Comment:
- [OPTIONAL] Please use the image as the background not the main content of your header. Because the image that you used is just for decoration not as the content of the page which will not help the user any how.

Reply:
@skyv26 I'd pass on this.

Please can you knindly review

Reply:
@donice Please submit your review any other reviewer will soon give you approval if you made the changes I mentioned.

Reply:
@donice I can see that your path is working but there is no license content in your `MIT.md` file. Please add the content in your file.

Reply:
@donice Well done, Good Job 👍🏻

Reply:
@skyv26 I have fixed this. Please kindly revew

Reply:
@skyv26 @Owusu-Desmond would I have to request another review on the dashboard

Reply:
@donice Yes, please submit for review on dashboard.